### PR TITLE
Update README and simplify files for first time Fleet users

### PR DESCRIPTION
**Pull request by noahtalerman**
Note: Wait to merge this PR until the GitOps [reference docs PR](https://github.com/fleetdm/fleet/pull/19740) is merged.

- Remove beta call out
- Guide user on how to add required environment variables
- Point users to reference documentation
- Remove everything but what's required + starter scripts, policies, and queries in `default.yml` and teams files. Make it feel easy for first time users

### CODE REVIEW

File: default.yml [hunk 1]
```diff
@@ -1,58 +1,16 @@
...
-      - path: ./lib/windows-screenlock.xml
-  scripts:
-    - path: ./lib/collect-fleetd-logs.sh
-queries:
-  - path: ./lib/collect-fleetd-update-channels.queries.yml
 policies:
+queries:
 agent_options:
-  path: ./lib/agent-options.yml
+- path: ../lib/agent-options.yml
```
Comment (getvictor) [inline default.yml:8]:
```suggestion
  path: ./lib/agent-options.yml
```

**Review by getvictor — COMMENTED** [review]
Left a comment

File: default.yml (hunk 1 above)
Comment (noahtalerman) [reply default.yml]: Thanks. Good catch

**Review by getvictor — APPROVED** [review]
LGTM

**Pull request merged**